diff --git a/src/lib.rs b/src/lib.rs
index 49cb2b7..3d3cf8b 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -42,6 +42,7 @@ mod clipboard;
 #[cfg(not(any(target_os = "android", target_os = "ios")))]
 pub mod core_main;
 mod custom_server;
+pub mod embedded_config;
 mod lang;
 #[cfg(not(any(target_os = "android", target_os = "ios")))]
 mod port_forward;
diff --git a/src/core_main.rs b/src/core_main.rs
index 6b437a9..8c8cf15 100644
--- a/src/core_main.rs
+++ b/src/core_main.rs
@@ -33,6 +33,7 @@ pub fn core_main() -> Option<Vec<String>> {
         return None;
     }
     crate::load_custom_client();
+    crate::embedded_config::init();
     #[cfg(windows)]
     if !crate::platform::windows::bootstrap() {
         // return None to terminate the process
